langchain/embeddings/base.py

```diff
@@ -13,3 +13,7 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
     @abstractmethod
     def embed_query(self, text: str) -> List[float]:
         """Embed query text."""
+
+
+# For backwards compatibility.
+Embedding = EmbeddingModel
```

Comment:
Ah good!

Reply:
could even give this an init that logs deprecation warning